### PR TITLE
silence PD alerts notice

### PR DESCRIPTION
**Pull request by aweiteka**
Related to https://github.com/openshift/managed-notifications/pull/8

This may be rare but we needed it once. Might as well include it here in case we see it again.

### CODE REVIEW

File: osd/silence_namespace_alerts_notice.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Warning",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "NOTICE: Silencing alerts for 'openshift-*' user namespaces",
+    "description": "The Red Hat SRE team monitors all OpenShift Dedicated namespaces matching pattern 'openshift-*'. This namespace prefix is reserved for infrastructure use. Red Hat SRE does not support monitoring user namespaces matching this pattern. Alerts for namespace '${NAMESPACE}' have been silenced and will not be re-enabled. Cluster metrics and alerts are accessed from the cluster console.",
```
Comment (mmazur) [inline osd/silence_namespace_alerts_notice.json:6]: > Cluster metrics and alerts are accessed from the cluster console.

I'm confused on the point of this sentence.

Comment (aweiteka) [reply osd/silence_namespace_alerts_notice.json:6]: I'm pointing the customer to the cluster's prometheus/alertmanager/grafana consoles. dedicated-admins have access, correct?

Comment (fahlmant) [reply osd/silence_namespace_alerts_notice.json:6]: Correct

Comment (mmazur) [reply osd/silence_namespace_alerts_notice.json:6]: Oh, so this is just informative that they _can be_ accessed. I'm not sure this is super clear.

"Cluster metrics and alerts can still be accessed from the cluster console." to make it clear this is supposed to be "reassuring"?

Comment (aweiteka) [reply osd/silence_namespace_alerts_notice.json]: ACK. updated

**Pull request merged**